src/js/edu-benefits/utils/helpers.jsx

```diff
@@ -157,3 +158,8 @@ export function showYesNo(field) {
 export function enumToNames(enumValues, names) {
   return enumValues.map(item => names[item]);
 }
+
+export const stateLabels = states.USA.reduce((current, { label, value }) => {
```

Comment:
I don't think so. Those two transform object -> object, this is going from array -> object.